src/main/scala/centaur/api/CromwellClient.scala

```diff
@@ -33,6 +35,19 @@ object CromwellClient {
   final implicit val materializer: ActorMaterializer = ActorMaterializer(ActorMaterializerSettings(system))
   final val apiPath = "/api/workflows/v1"
 
+  def decodeResponse(response: HttpResponse): HttpResponse = {
+    val decoder = response.encoding match {
+      case HttpEncodings.gzip ⇒
+        Gzip
+      case HttpEncodings.deflate ⇒
+        Deflate
```

Comment:
Three TOLs for the price of 1!

Cromwell should probably be consistent in which one of these it uses, and the choice of compression should be written down as part of the API spec.

Is there a choice whereby Cromwell sometimes doesn't compress?

Would it be possible to have an alternate endpoints for compression (e.g. raw: `api/v1/metadata`, gzip: `api/v1/metadata/gzip`, gzip: deflate: `api/v1/metadata/deflate`, ...)